tests/foreman/ui/test_contentview.py

```diff
@@ -7,9 +7,9 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Component
 
-:CaseComponent: UI
+:CaseComponent: ContentView
```

Comment:
Bugzilla component is called `ContentViews`, in plural. See testimony.yaml for reference.